src/fairseq2/models/conformer/convolution.py

```diff
@@ -6,6 +6,7 @@
 
 from typing import Optional
 
+import torch.nn.functional as F
```

Comment:
Recently I have started using explicit imports (and updated the code base to follow that convention), so in the rest of the code base we use `from torch.nn.functional import pad`. I personally prefer this style since it gives you immediately the overview of that features/ops we use at the top of the file.